web/.env.local

```diff
@@ -0,0 +1,7 @@
+REEARTH_WEB_DEVELOPER_MODE=
+REEARTH_WEB_CESIUM_ION_TOKEN_URL=
```

Comment:
```suggestion
```
This one is more for us only, it's not needed for OSS i think.

Reply:
Okay, I have indicated which ones are required and which ones are optional